payments/models.py

```diff
@@ -119,6 +119,18 @@ def get_queryset(self):
                         ))
 
 
+class PosterOption(models.Model):
+    workshop = models.ForeignKey(Workshop)
+    name = models.CharField(max_length=300)
+    sort_order = models.IntegerField()
```

Comment:
Should we provide a help text for this? Had to look at the code to see what was expected when creating a test workshop.